frontend-react/src/components/FileHandlers/FileHandlerMessaging.test.tsx

```diff
@@ -93,7 +93,7 @@ describe("FileErrorDisplay", () => {
             field: "first field",
             trackingIds: ["first_id"],
             scope: "unclear",
-            errorCode: "INVALID_HL7_MESSAGE_DATE_VALIDATION",
+            errorCode: "INVALID_MSG_PARSE_DATE",
```

Comment:
I changed the names of existing errorCodes so I had to update the old names where used.

Reply:
I think another improvement we could make here is adding enums for each of the possible error codes now that they're settled on since it doesn't look like we maintain them outside of the incomplete ErrorCodeTranslation.

This is out of the scope of this pull request, though, so I'll add a fast-follow Experience ticket -- might also easier to handle with the types generation too!

Reply:
Yeah, I don't see why not!  But we're still in limbo on where that plugin should live so it might be a while 😬 

(cc @jpandersen87 for visibility)

Reply:
Theoretically, if the plugin was in use and a platform engineer tagged a class that had a property containing an error code enum we would get that enum in the output.